### ISSUE TITLE
Best Practices

### ISSUE THREAD

**Issue by oldoc63** [oldoc63/learningFS#903]

**Comment by oldoc63** [comment]
1. Let’s get started by adding the word search to the search button. This will make this feature more obvious to users with low vision, more accessible to people using screen readers, and it will also save the function of our site if that icon ever fails to load! In index.html, add the word Search after this piece of code: <i class="fa fa-search"></i>. For more on what this means, see the hint!

**Comment by oldoc63** [comment]
Review

You made the cupcake recipe site look much better! You practiced some new CSS syntax, and you learned the power of visual contrast, for both overall design, and for accessibility.

You gained some knowledge on web accessibility and how to make your websites usable for all users.

Remember, the keys to building a more visually inclusive Web are:

    Using contrast so our text and colors provide better visual cues
    Using correct font size so our content is legible
    Creating user interfaces that are enhanced by, but not dependent on, color

Following these guidelines can increase the usability of our websites. In the next lesson, you will learn semantic HTML tags, which improve the accessibility of sites for people using a screen reader.
